pkg/agent/cniserver/server_linux.go

```diff
@@ -0,0 +1,24 @@
+// +build !windows
```

Comment:
Why not "+build linux"? Does this file also need to run on other platforms like darwin?

Reply:
I think "+build !windows" can be removed, we only support linux now.